target/linux/bmips/Makefile

```diff
@@ -0,0 +1,19 @@
+include $(TOPDIR)/rules.mk
+
+ARCH:=mips
+BOARD:=bmips
+BOARDNAME:=Broadcom BMIPS
+CPU_TYPE:=mips32
+FEATURES:=squashfs usb pci
+KERNEL_PATCHVER:=5.4
```

Comment:
Consider adding an empty line before so KERNEL_PATCHVER is separate from the block.